### ISSUE TITLE
oauth okta.networkZones support

### ISSUE THREAD

**Issue by cvirtucio** [okta/terraform-provider-okta#1493]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The provider's [list of valid scopes](https://github.com/okta/terraform-provider-okta/blob/d0c5948d850d32ae733e756631f198a7602da0a9/okta/resource_okta_app_oauth_api_scope.go#L14-L40) currently does not support the `okta.networkZones.*` scopes, listed [here](https://developer.okta.com/docs/api/oauth2/). This results in an error whenever we try to add it in:

```txt
Error: expected value to be one of 'okta.apps.manage', 'okta.apps.read', 'okta.authorizationServers.manage', 'okta.authorizationServers.read', 'okta.authenticators.manage', 'okta.authenticators.read', 'okta.brands.manage', 'okta.brands.read', 'okta.captchas.manage', 'okta.captchas.read', 'okta.clients.manage', 'okta.clients.read', 'okta.clients.register', 'okta.devices.manage', 'okta.devices.read', 'okta.domains.manage', 'okta.domains.read', 'okta.eventHooks.manage', 'okta.eventHooks.read', 'okta.events.read', 'okta.factors.manage', 'okta.factors.read', 'okta.groups.manage', 'okta.groups.read', 'okta.idps.manage', 'okta.idps.read', 'okta.inlineHooks.manage', 'okta.inlineHooks.read', 'okta.linkedObjects.manage', 'okta.linkedObjects.read', 'okta.logs.read', 'okta.policies.manage', 'okta.policies.read', 'okta.profileMappings.manage', 'okta.profileMappings.read', 'okta.roles.manage', 'okta.roles.read', 'okta.schemas.manage', 'okta.schemas.read', 'okta.sessions.manage', 'okta.sessions.read', 'okta.templates.manage', 'okta.templates.read', 'okta.trustedOrigins.manage', 'okta.trustedOrigins.read', 'okta.users.manage', 'okta.users.read', 'okta.users.manage.self', 'okta.users.read.self', 'okta.userTypes.manage', 'okta.userTypes.read', got 'okta.networkZones.manage'
```

### New or Affected Resource(s)

- okta_app_oauth

### Potential Terraform Configuration

```hcl
resource "okta_app_oauth_api_scope" "terraform_identity_service_client" {
  scopes = ["okta.networkZones.manage"]
}
```

### References

- #0000

**Comment by awagneratzendesk** [comment]
Would probably be good to do a run through of all the Oauth Scopes now available in https://developer.okta.com/docs/api/oauth2/ and sync the provider to that list?

**Comment by awagneratzendesk** [comment]
I'm going to PR this shortly after I do some local testing

**Comment by monde** [comment]
I will be taking @awagneratzendesk's PR #1494

**Comment by monde** [comment]
This will be fixed in the next release from #1494
